docs/examples/general/data_loading/numpy_reader.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.                                shard_id=0, num_shards=1)
</pre><p>Do we really want to put that here? We haven't said anything about sharding here and I believe this topic is a bit out of scope of this tutorial.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/3095/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>